docs/docs/reference/other-new-features/export.md

```diff
@@ -50,8 +49,9 @@ of one of the following forms:
  - A _renaming selector_ `x => y` creates aliases for all eligible members of `path` that are named `x`, but the alias is named `y` instead of `x`.
  - An _omitting selector_ `x => _` prevents `x` from being aliased by a subsequent
    wildcard selector.
- - A _wildcard selector_ creates aliases for all eligible members of `path` except for
-   synthetic members generated by the compiler and those members that are named by a previous simple, renaming, or omitting selector.
+ - A _given selector_ `given x` has an optional type bound `x`. It creates aliases for all eligable given instances that conform to either `x`, or `Any` if `x` is omitted, except for members that are named by a previous simple, renaming, or omitting selector.
+ - A _wildcard selector_ `_` creates aliases for all eligible members of `path` except for given instances,
+   synthetic members generated by the compiler (unless there is an eligable member sharing the same name that is not synthetic) and those members that are named by a previous simple, renaming, or omitting selector.
```

Comment:
typo: eligable. But I believe the whole (unless ... ) clause is more confusing than informative. I think we can safely drop it.